python/triqs_dft_tools/converters/plovasp/proj_shell.py

```diff
@@ -277,7 +261,8 @@ def select_projectors(self, ib_win, ib_min, ib_max):
         self.ib_win = ib_win
         self.ib_min = ib_min
         self.ib_max = ib_max
-        nb_max = ib_max - ib_min + 1
+        nb_max = ib_max - ib_min + 1    
+        print("nb_max : {}".format(nb_max))
```

Comment:
remove debug print?

Reply:
I am not sure these were written by me and not already present in a previous version, but in any case can be removed